### PR TITLE
Add liquidation method

### PR DESCRIPTION
**Pull request by eskrano**
Add liquidation method

### CODE REVIEW

**Comment by zhouaini528** [comment]
OK. Sorry, I will reply to you now. But I need a little time, I am currently developing an API that combines multiple exchanges, and it will be great soon. The extension of bitmex is very good. If you are in a hurry, you can expand it yourself.

File: src/Api/Liquidation.php [hunk 1]
```diff
@@ -5,5 +5,12 @@
...
-
-}
\ No newline at end of file
+    public function get($data)
+    {
+        $this->type='GET';
+        $this->path='/api/v1/liquidation';
+        $this->data=$data;
+        
+        return $this-exec();
```
Comment (zhouaini528) [inline src/Api/Liquidation.php]: return $this->exec();

**Comment by eskrano** [comment]
> OK. Sorry, I will reply to you now. But I need a little time, I am currently developing an API that combines multiple exchanges, and it will be great soon. The extension of bitmex is very good. If you are in a hurry, you can expand it yourself.

 i fork your repo and add some methods. https://github.com/eskrano/bitmex-php

**Comment by zhouaini528** [comment]
> i fork your repo and add some methods. https://github.com/eskrano/bitmex-php

I've merged it.Welcome to improve it with me

**Pull request merged**